solr/solrj/src/test/org/apache/solr/client/solrj/impl/ClusterStateProviderTest.java

```diff
@@ -51,6 +54,15 @@ public static void setupCluster() throws Exception {
                 .resolve("streaming")
                 .resolve("conf"))
         .configure();
+    cluster.waitForAllNodes(30);
+    System.setProperty("solr.solrj.cache.timeout.sec", "1");
```

Comment:
(am random pull request browser/scanner only here) -- maybe have matching `System.clearProperty` during test cleanup to avoid carry-ever effects between tests.

Reply:
Christine, Solr's test infrastructure auto-restores all system properties to as they were from test suite to test suite thanks to SystemPropertiesRestoreRule.  But not test-method to test-method, albeit that's not pertinent for the line you commented (as it's in a BeforeClass).